src/ui/public/elasticsearch_errors/is_term_size_zero_error.js

```diff
@@ -1,6 +1,10 @@
 import ElasticsearchError from './elasticsearch_error';
 
 export default function isTermSizeZeroError(error) {
-  const esError = new ElasticsearchError(error);
-  return esError.hasRootCause('size must be positive, got 0');
+  try {
+    const esError = new ElasticsearchError(error);
+    return esError.hasRootCause('size must be positive, got 0');
+  } catch (err) {
+    return false;
+  }
```

Comment:
Created https://github.com/elastic/kibana/issues/8516 to discuss addressing the underlying design of this module, but I'd like to move forward with the implementation as is so that we can get 5.0.0-rc1 out the door.